flashcards/backend/fixtures/example_data.json

```diff
@@ -0,0 +1,624 @@
+    "fields": {
+      "name": "Can change deck",
+      "content_type": 10,
+      "codename": "change_deck"
+    }
+  },
+  {
+    "model": "auth.permission",
+    "pk": 39,
+    "fields": {
+      "name": "Can delete deck",
+      "content_type": 10,
+      "codename": "delete_deck"
+    }
+  },
+  {
+    "model": "auth.permission",
+    "pk": 40,
+    "fields": {
+      "name": "Can view deck",
+      "content_type": 10,
+      "codename": "view_deck"
+    }
+  },
+  {
+    "model": "contenttypes.contenttype",
+    "pk": 1,
+    "fields": {
+      "app_label": "admin",
+      "model": "logentry"
+    }
+  },
+  {
+    "model": "contenttypes.contenttype",
+    "pk": 2,
+    "fields": {
+      "app_label": "auth",
+      "model": "permission"
+    }
+  },
+  {
+    "model": "contenttypes.contenttype",
+    "pk": 3,
+    "fields": {
+      "app_label": "auth",
+      "model": "group"
+    }
+  },
+  {
+    "model": "contenttypes.contenttype",
+    "pk": 4,
+    "fields": {
+      "app_label": "contenttypes",
+      "model": "contenttype"
+    }
+  },
+  {
+    "model": "contenttypes.contenttype",
+    "pk": 5,
+    "fields": {
+      "app_label": "sessions",
+      "model": "session"
+    }
+  },
+  {
+    "model": "contenttypes.contenttype",
+    "pk": 6,
+    "fields": {
+      "app_label": "token_blacklist",
+      "model": "blacklistedtoken"
+    }
+  },
+  {
+    "model": "contenttypes.contenttype",
+    "pk": 7,
+    "fields": {
+      "app_label": "token_blacklist",
+      "model": "outstandingtoken"
+    }
+  },
+  {
+    "model": "contenttypes.contenttype",
+    "pk": 8,
+    "fields": {
+      "app_label": "users",
+      "model": "customuser"
+    }
+  },
+  {
+    "model": "contenttypes.contenttype",
+    "pk": 9,
+    "fields": {
+      "app_label": "decks",
+      "model": "category"
+    }
+  },
+  {
+    "model": "contenttypes.contenttype",
+    "pk": 10,
+    "fields": {
+      "app_label": "decks",
+      "model": "deck"
+    }
+  },
+  {
+    "model": "sessions.session",
+    "pk": "ctgb3jkpo2l4yscm4jhcwc2ynijytcaq",
+    "fields": {
+      "session_data": ".eJxVjEsOgkAMQO8ya5nAtDLFpXvPQDptR1ADCZ-V8e5CwkLX7_N2La9L166zTW2v7uJMQABqKJgaLjATFYSAhcUKEJMqI7rTb5ZYnjbsrT54uI9exmGZ-uR3xR909rdR7XU93L9Bx3O31ZWUcJYMGjgEApNEWaJYyjWrYSmlJtKNhdyQBWgsZOJolQGFaNF9vgc2Qgo:1pii2T:9M8_chfMGMpPPsTPTkBmORsls1l-ixEE3SU0oCfFW2o",
+      "expire_date": "2023-04-15T20:41:29.916Z"
+    }
+  },
+  {
+    "model": "users.customuser",
+    "pk": "ec3c3363-a89a-4f88-8434-e71344bdda44",
+    "fields": {
+      "password": "pbkdf2_sha256$390000$TUN67aki3esbGjy2JaxrYK$WiSIMu9v/Ix8VEGJBtqRWABaWMaMKYSP3yFxMm+Z9n0=",
+      "last_login": "2023-04-01T20:41:29.898Z",
+      "is_superuser": true,
+      "first_name": "",
+      "last_name": "",
+      "is_staff": true,
+      "is_active": true,
+      "date_joined": "2023-04-01T20:40:46.700Z",
+      "email": "a@a.pl",
+      "username": "aa",
+      "groups": [],
+      "user_permissions": []
+    }
+  },
+  {
```

Comment:
it's good that you prepared the example data, but I think to this point you can remove, things. The above are from the migration, Paweł on the course always removed the above data

Reply:
how to then resolve problem of uuid which is unique for user. You user and my user will have different hash. During course we were using user_pk which always was 1 and is common for you and my user?